src/main/java/org/terasology/projsndwv/components/ProcessingComponent.java

```diff
@@ -18,14 +18,15 @@
 import org.terasology.entitySystem.Component;
 
 /***
- * Indicates to ApiaryScreen that bees are currently mating in an apiary, and at what game time they will finish.
+ * Indicates to screens that processes are occuring in an apiary, extractor, or injector, and at what game time they will
+ * complete.
  */
-public final class ApiaryMatingComponent implements Component {
-    public long mateFinishTime;
+public final class ProcessingComponent implements Component {
```

Comment:
I dunno about this rename - that's an oddly generic new name, while the prior was nice and explicit. Not that the risk of component name conflicts is great yet. Or are you intending for this to be used beyond just bee stuff?

Reply:
Couldn't think of a good, precise name. Considering its use could reasonably be even broader than "processing," that's what I went with.